defi-pulse/src/symbols/directory.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
We need to support multiple different networks here. For example, let's say we want to support this functionality on BSC & Matic networks. Rename it as `directory.mainnet.json` and add a hardcoded `network = 'mainnet'` param to the top-level file. Probably something like this:

```javascript
const index = await makeIndex(components, units, 'mainnet')
```